test/unit/mock/cmd/find-nginx-confd-example-conf

```diff
@@ -0,0 +1 @@
+/etc/nginx/conf.d/example.conf
```

Comment:
For completeness of testing, I'd change this to return multiple files to ensure the code actually iterates over each of them.